lib/reek/cli/command/todo_list_command.rb

```diff
@@ -36,13 +36,13 @@ def scan_for_smells
         end
 
         def groups_for(smells)
-          @groups ||= begin
-            Hash[
-              smells.group_by(&:smell_type).map do |smell_type, smells_for_type|
-                [smell_type, { 'exclude' => smells_for_type.map(&:context) }]
+          @groups ||=
+            begin
+              todos = smells.group_by(&:smell_class).map do |smell_class, smells_for_class|
```

Comment:
>>  Providing a #to_yaml that returns the smell type string might work, but it's a can of worms I'm not that eager to open.

I don't follow - like what?

>> Use the string. This is possible if we can fetch the corresponding detector class from a smell repository object. I would prefer this since it will allow decoupling the smell warning from the smell detector in the end. Also, it may allow us to use one smell repository shared by all examiners.

👍  - but I'd still like to hear more about the can of worms above ;)